man/dnstable_query.3

```diff
@@ -84,6 +84,12 @@ dnstable_query_set_aggregated(struct dnstable_query *\fR\fB\fIq\fR\fR\fB,
 .fi
 .sp
 .nf
+\fBvoid
```

Comment:
This is missing its corresponding line in the DESCRIPTION section below (under "dnstable_query_set_...() functions")